src/presentation/controller/registration/PageRegistrationType.ts

```diff
@@ -0,0 +1,9 @@
+enum PageRegistrationType {
```

Comment:
As discussed... just name this as PageType as already in a 'registration' folder. Same applies to the `RegistrationController`.